cam2/src/main/java/com/commonsware/cwac/cam2/helper/ImageHelper.java

```diff
@@ -0,0 +1,42 @@
+package com.commonsware.cwac.cam2.helper;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.util.SparseArray;
+
+import com.google.android.gms.vision.Frame;
+import com.google.android.gms.vision.face.Face;
+import com.google.android.gms.vision.face.FaceDetector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ImageHelper {
+  public boolean isFaceOccupancy(Context context, Bitmap bitmap, float percent) {
```

Comment:
rename isFaceOccupancy to isFacePresentWithMinimumOccupancy
rename percent to minimumOccupancyPercentage